packages/common/base/src/apiClient/CrossmintApiClient.ts

```diff
@@ -42,8 +42,10 @@ export class CrossmintApiClient extends ApiClient {
         return ApiClient.normalizePath(baseUrl);
     }
 
-    get authHeaders() {
+    get commonHeaders() {
         return {
+            "x-sdk-name": this.internalConfig.sdkMetadata.name,
+            "x-sdk-version": this.internalConfig.sdkMetadata.version,
```

Comment:
i think we called these x-client-name, x-client-version in other places no?

Reply:
cant find it in the code but see it referenced here

https://github.com/Crossmint/crossmint-sdk/issues/18